lnd_test.go

```diff
@@ -2005,10 +2005,14 @@ func TestLightningNetworkDaemon(t *testing.T) {
 	}
 
 	// With the btcd harness created, we can now complete the
-	// initialization of the network. args - list of lnd arguments,
-	// example: "--debuglevel=debug"
+	// initialization of the network. Run nodes with info debug mode:
+	lndArgs := []string{
+		"--debuglevel=info",
```

Comment:
No, it just so annoying every time type it in `lnd_test.go`, so I thought it might be forth to just place `info` by default and change to `debug` when it will be needed.